katsdptelstate/test/test_rdb_handling.py

```diff
@@ -150,7 +150,7 @@ def test_zset_with_big_entry(self) -> None:
     def _test_hash(self, items: Mapping[bytes, bytes]) -> None:
         # redis-py 3.5 implements multi-item hset and deprecates hmset.
         # Use whichever version will work and avoid a warning.
-        if redis.__version__ >= LooseVersion('3.5'):     # type: ignore
+        if Version(redis.__version__) >= Version('3.5'):     # type: ignore
```

Comment:
Since you now have `redis>=4.2` as a requirement, you can just delete this branch, and avoid adding a dependency on `packaging`.

Reply:
🤦🏼‍♂️ I somehow remembered the 3.3 even while I was moving beyond it to 4.2. Thanks for picking that up!